### PR TITLE
DSC-1310 adding ingress annotations

### PR DESCRIPTION
**Pull request by michalnawrockidj**

### CODE REVIEW

**Review by andrewrlee — APPROVED** [review]
LGTM

**Review by andrewrlee — CHANGES_REQUESTED** [review]
Not sure this is right

File: helm_deploy/hmpps-welcome-people-into-prison-api/values.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ generic-service:
     host: app-hostname.local    # override per environment
     tlsSecretName: hmpps-welcome-to-prison-cert
     path: /
+    external-dns.alpha.kubernetes.io/aws-weight: "100"
```
Comment (andrewrlee) [inline helm_deploy/hmpps-welcome-people-into-prison-api/values.yaml:17]: this needs to be inside an annotations tag

Comment (petergphillips) [reply helm_deploy/hmpps-welcome-people-into-prison-api/values.yaml:17]: you're using the `generic-service` - there is no need for this annotation as it is added automatically if you grab the latest version of the chart

Comment (michalnawrockidj) [reply helm_deploy/hmpps-welcome-people-into-prison-api/values.yaml]: Good point.
I updated as us suggested. I removed `path: /` as this is a part of the newest version as well

**Review by petergphillips — APPROVED** [review]
LGTM

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**